tools/flaky-test-reporter/result.go

```diff
@@ -165,3 +166,23 @@ func collectTestResultsForRepo(jc *JobConfig) (*RepoData, error) {
 	}
 	return rd, nil
 }
+
+// getLatestFinishedBuilds is an inexpensive way of listing latest finished builds, in comparing to
+// the GetLatestBuilds function from prow package, as it doesn't precompute start/finish time before sorting.
+// This function takes the assumption that build IDs are always incremental integers
```

Comment:
This is true if you see the implementation of the ID used by Prow for the jobs.

Reply:
that's right
https://github.com/kubernetes/test-infra/blob/master/prow/pjutil/tot.go#L80